code-in-progress/gg_plot_etiology_strat.R

```diff
@@ -0,0 +1,124 @@
+
+##----------------------------
```

Comment:
@realirena this folder 'code-in-progress' will not make it into the final package. So can I assume code in this folder is not considered part of a real pull request?

Reply:
Yes, removed it for now! In the future maybe we could add some ggplot features.